python/ml_wrappers/dataset/dataset_wrapper.py

```diff
@@ -45,11 +45,11 @@ def __init__(self, dataset, clear_references=False):
         :param clear_references: A memory optimization that clears all references after use in explainers.
         :type clear_references: bool
         """
-        if (not isinstance(dataset, pd.DataFrame) and not isinstance(dataset, pd.Series) and
-                not isinstance(dataset, np.ndarray) and not issparse(dataset) and
-                not str(type(dataset)).endswith(".DenseData'>") and
-                not str(type(dataset)).endswith("torch.Tensor'>") and
-                not str(type(dataset)).endswith("BatchDataset'>")):
+        if not isinstance(dataset, pd.DataFrame) and not isinstance(dataset, pd.Series) and \
+                not isinstance(dataset, np.ndarray) and not issparse(dataset) and \
+                not str(type(dataset)).endswith(".DenseData'>") and \
+                not str(type(dataset)).endswith("torch.Tensor'>") and \
+                not str(type(dataset)).endswith("BatchDataset'>"):
             raise TypeError("Got type {0} which is not not supported in DatasetWrapper".format(
```

Comment:
nit: fix type error message here
not not supported -> not supported

Reply:
done